### PR TITLE
Stubbhult 1.0

### PR DESCRIPTION
**Pull request by Facetti**
Tested, but not all side missions. 
Wiki will be updated seperately.

### CODE REVIEW

File: dom_maker/make_dom.bat [hunk 1]
```diff
@@ -1,4 +1,4 @@
-rem Domination Build Batch, really shitty batch file :D
+jrem Domination Build Batch, really shitty batch file :D
```
Comment (longtimegamer) [inline dom_maker/make_dom.bat:1]: jrem? typo?

Comment (Facetti) [reply dom_maker/make_dom.bat:1]: arghh - yes.

**Comment by Facetti** [comment]
port finished, but not all side missions tested.

File: co30_Domination.Altis/x_macros.sqf [hunk 2]
```diff
@@ -2,13 +2,22 @@
...
-#define __TRACE_1(tmsg,parame1) diag_log format ["T%1,DT%2,F%3,FPS%4,%5,%6,'%7',%8: %9", time, diag_tickTime, diag_frameno, diag_fps, call {_xarsx = __FILE__ splitString "\";_xarsx select (count _xarsx - 1)}, __LINE__, tmsg, parame1, call compile format ["%1", parame1]];
-#define __TRACE_2(tmsg,parame1,parame2) diag_log format ["T%1,DT%2,F%3,FPS%4,%5,%6,'%7',%8: %9,%10: %11", time, diag_tickTime, diag_frameno, diag_fps, call {_xarsx = __FILE__ splitString "\";_xarsx select (count _xarsx - 1)}, __LINE__, tmsg, parame1, call compile format ["%1", parame1], parame2, call compile format ["%1", parame2]];
-#define __TRACE_3(tmsg,parame1,parame2,parame3) diag_log format ["T%1,DT%2,F%3,FPS%4,%5,%6,'%7',%8: %9,%10: %11,%12: %13", time, diag_tickTime, diag_frameno, diag_fps, call {_xarsx = __FILE__ splitString "\";_xarsx select (count _xarsx - 1)}, __LINE__, tmsg, parame1, call compile format ["%1", parame1], parame2, call compile format ["%1", parame2], parame3, call compile format ["%1", parame3]];
+
```
Comment (longtimegamer) [inline co30_Domination.Altis/x_macros.sqf:5]: what is this part?

File: co30_Domination.Altis/missions/m/MissionDescription.txt [hunk 3]
```diff
@@ -0,0 +1,141 @@
+fn_create_sm_bymarkers.sqf
```
Comment (longtimegamer) [inline co30_Domination.Altis/missions/m/MissionDescription.txt:1]: what is this file?

**Comment by longtimegamer** [comment]
looks fine to me, however Xeno has mentioned there are no more releases for Domination in Arma 3 except for emergency fixes

File: co30_Domination.Altis/x_macros.sqf (hunk 2 above)
Comment (Facetti) [reply co30_Domination.Altis/x_macros.sqf:5]: No functional impact.
while I tried to understand the code and the dependencies, I've reduced the TRACE output for easier reading.
Original code is not deleted, just commented.

**Comment by Facetti** [comment]
> looks mostly fine to me. can you explain the x_macros.sqf changes?
> 
> however Xeno has mentioned there are no more releases for Domination in Arma 3 except for emergency fixes

...but not supporting Stubbhult is an emergency - and needs a fix ;-)
anyway, feel free to link to the fork https://github.com/Facetti/Domination for getting the latest release or to the PBO https://github.com/Facetti/Domination/releases/download/v1.1/co40_domination_4_70_blufor.stubbhult.pbo.

**Comment by Xeno69** [comment]
Like longtimegamer wrote, 4.70 is the last Dom A3 release. Feel free to release a Domi version with your changes.

**Pull request closed without merging**